lms/djangoapps/verify_student/api.py

```diff
@@ -33,3 +40,63 @@ def send_approval_email(attempt):
     else:
         email_context = {'user': attempt.user, 'expiration_datetime': expiration_datetime.strftime("%m/%d/%Y")}
         send_verification_approved_email(context=email_context)
+
+
+def create_verification_attempt(user, name, status, expiration_datetime=None):
+    """
+    Create a verification attempt.
+
+    This method is intended to be used by IDV implementation plugins to create VerificationAttempt instances.
+
+    Args:
+        user (User): the user (usually a learner) performing the verification attempt
+        name (string): the name being ID verified
+        status (string): the initial status of the verification attempt
+        expiration_datetime (datetime, optional): When the verification attempt expires. Defaults to None.
+
+    Returns:
+        id (int): The id of the created VerificationAttempt instance
+    """
+    verification_attempt = VerificationAttempt.objects.create(
+        user=user,
+        name=name,
+        status=status,
+        expiration_datetime=expiration_datetime,
+    )
+
+    return verification_attempt.id
+
+
+def update_verification_attempt_status(attempt_id, status):
+    """
+    Update the status of a verification attempt.
+
+    This method is intended to be used by IDV implementation plugins to update VerificationAttempt instances.
+
+    Arguments:
+        * id (str): the verification attempt id of the attempt to update
+        * status (str): the new status
+
+    Returns:
+        * None
+    """
+    status_list = [attr for attr in dir(VerificationAttemptStatus) if not attr.startswith('__')]
+
+    if status not in status_list:
+        log.error(
+            f'update_verification_attempt_status called with invalid status. '
+            f'Status must be one of: {status_list}',
+        )
+        raise VerificationAttemptInvalidStatus
+
+    try:
+        attempt = VerificationAttempt.objects.get(id=attempt_id)
+    except VerificationAttempt.DoesNotExist:
+        log.error(
+            f'VerificationAttempt with id {attempt_id} was not found '
+            f'when updating the attempt to status={status}',
+        )
+        raise VerificationAttempt.DoesNotExist  # pylint: disable=raise-missing-from
```

Comment:
If you're simply re-raising the same exception you caught, then it's best practice to just use `raise`. This will do what you're doing here and will avoid the pylint error, so you could remove the disable line.